ise_named_groups.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
dummy file so that the code doesn't produce errors.  It gets reloaded every 30 seconds